README.md

```diff
@@ -12,3 +12,662 @@ Public CDN files:
 - [kinto-client.js](https://npmcdn.com/kinto-client/dist/kinto-client.js)
 - [kinto-client.min.js](https://npmcdn.com/kinto-client/dist/kinto-client.min.js)
 - [kinto-client.noshim.js](https://npmcdn.com/kinto-client/dist/kinto-client.noshim.js)
+
+## Usage
+
+A client instance is created using the `KintoClient` constructor, passing it the remote Kinto server URL:
+
+```js
+const client = new KintoClient("https://kinto.dev.mozaws.net/v1/");
+```
+
+## Buckets
+
+### Listing buckets
+
+```js
+client.listBuckets();
+```
+
+Sample result:
+
+```js
+[
+  {
+    id: "comments",
+    last_modified: 1456182233221,
+  },
+  {
+    id: "blog",
+    last_modified: 1456181213214,
+  },
+]
+```
+
+### Creating a new bucket
+
+```js
+client.createBucket("blog");
+```
+
+Sample result:
+
+```js
+{
+  "data": {
+    "last_modified": 1456182233221,
+    "id": "foo"
+  },
+  "permissions": {
+    "write": [
+      "basicauth:0f7c1b72cdc89b9d42a2d48d5f0b291a1e8afd408cc38a2197cdf508269cecc8"
+    ]
+  }
+}
+```
+
+### Selecting a bucket
+
+```js
+client.bucket("blog");
+```
+
+### Getting bucket permissions
+
+```js
+client.bucket("blog").getPermissions();
+```
+
+Sample result:
+
+```js
+{
+  "write": [
+    "basicauth:0f7c1b72cdc89b9d42a2d48d5f0b291a1e8afd408cc38a2197cdf508269cecc8"
+  ]
+}
+```
+
+### Setting bucket permissions
+
+#### Write permissions
+
+```js
+client.bucket("blog").setPermissions("write", ["github:john", "github:bob"]);
```

Comment:
I'm in the process of rewriting this anyway, as it simply overrides everything else atm :D